tdx-tdcall/src/tdx.rs

```diff
@@ -282,3 +283,8 @@ pub fn tdvmcall_mapgpa(paddress: u64, length: usize) {
         length
     );
 }
+
+pub fn tdvmcall_getinfo() -> bool {
```

Comment:
this interface design is useless.

please follow spec to input R12, and output R11~R14.